Sem. 01/Pract. 01/Pract.01-2.cpp

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+
+int check(const int* nums, char s, int N) {
+	int count = 0;
+	for (int i = 0; i < N; i++) {
+		if (nums[i] > (int)s) count++;
```

Comment:
No need of casting

Reply:
I know, but our colleague Manolov said it`s okay if I add it, so that I`m absolutely sure that it will be cast. ;)